tests/platform_tests/api/test_psu.py

```diff
@@ -28,6 +28,7 @@
 ]
 
 STATUS_LED_COLOR_GREEN = "green"
+STATUS_LED_COLOR_ORANGE = "orange"
```

Comment:
Why do we need both `amber` and `orange`? They are essentially the same color.

Reply:
This is the idea of this PR, to group similar colors who share the same logical meaning.
Some platforms for example indicate 'fault' status as red but some use orange or amber, so to fit all in one test we need to have only one successful test from each of these groups.

Reply:
I understand this, but I'm curious why we need both `amber` and `orange`. Since they are essentially the same color, I feel we should choose only one, and all vendors should use that term to describe this color.

Reply:
I agree.
We can decide it will be "orange" and all other vendors will align to it?

Reply:
I believe all other vendors are using the term `amber`. Only Mellanox is using `orange`. It would be far less invasive if Mellanox aligned to `amber`.

Reply:
Fixed.